doc/administrator/galaxy/pipelines/assembly-annotation-collection/index.md

```diff
@@ -74,7 +74,7 @@ If everything was successful then all dependencies for this pipeline have been p
 [tbl2asn]: http://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/
 [Galaxy Main Shed]: http://toolshed.g2.bx.psu.edu/
 [IRIDA Toolshed]: https://irida.corefacility.ca/galaxy-shed
-[Assembly Annotation Galaxy Workflow]: ../test/assembly-annotation-collection/assembly-annotation-collection.ga
+[Assembly Annotation Collection Galaxy Workflow]: ../test/assembly-annotation-collection/assembly-annotation-collection.ga
```

Comment:
Why the collection workflow now?  Is it a different set of data to test with now?

Reply:
Do you mean why did I update this name to include **Collection**?  It was a mistake made previously since this is the documentation for the collection workflow, so the name of the link should reflect this.